Gopkg.lock

```diff
@@ -158,6 +158,20 @@
   pruneopts = "UT"
   revision = "e89373fe6b4a7413d7acd6da1725b83ef713e6e4"
 
+[[projects]]
```

Comment:
Your Gopkg.lock is out of date, please run a `dep ensure` to fix. (This file shouldn't be in the PR for a readme change.)